docs/installation.md

```diff
@@ -49,32 +49,3 @@ brew install flagd
 ```shell
 go install github.com/open-feature/flagd/flagd@latest
 ```
-
```

Comment:
+1 for removing this section

Reply:
I think we should include a cross-link in this installation page for providers

ex:- `Once flagd is installed, you can evaluate feature flags using your favorite language using flagd provider[Link to provider page] `

Reply:
Done.